README.md

```diff
@@ -78,16 +85,31 @@ The `LiveFixtures::Import` class allows you to specify the location of your fixt
       end
     end
 
+Options may be passed when initializing an importer as follow:
+ - show_progress: defaults to true.
+   Pass false to disable the progress bar output.
+ - skip_missing_tables: defaults to false.
+   Pass true to avoid raising an error when a table listed in insert_order has
+   no yml file.
+ - skip_missing_refs: defaults to true.
```

Comment:
Ah, okay. I am happy to change this - I had simply set the defaults to match behavior of version 0.2.0, but I think your rationale makes a lot of sense. Thanks for the review, and I'll change this and update the monolith accordingly.